### ISSUE TITLE
prettyChromoplot doesn't allow labelling of cytobands

### ISSUE THREAD

**Issue by kmcoyle** [morinlab/GAMBLR.viz#72]
Preserve similar functionality from maftools::gisticChromPlot -- label cytobands instead of (or with) genes.

**Comment by Kdreval** [comment]
if you just specify the cytoband in bed file that is used for labelling, I think it should plot it just like the gene :smile:

**Comment by kmcoyle** [comment]
True! I was looking to add functionality similar to maftools (i.e. annotate only if significant).

**Comment by Kdreval** [comment]
did you decide whether this needs improvement? I think the bed file approach works but unsure if this is still unresolved and you need help 😄
